h2/utilities.py

```diff
@@ -329,6 +329,13 @@ def _reject_pseudo_header_fields(headers, hdr_validation_flags):
             seen_pseudo_header_fields
         )
 
+    # If ':status' pseudo-header is not there in a response header, reject it
+    if (hdr_validation_flags.is_response_header and
+        not any(field in seen_pseudo_header_fields
+                for field in (b':status', u':status'))):
```

Comment:
Can we hoist this tuple up to the module scope? If we don't, CPython will recreate this tuple each time we enter the function, which is just unacceptable on a function that is called so frequently.